src/Microsoft.Extensions.SecretManager.Tools/Internal/SecretsStore.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.UserSecrets;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Extensions.SecretManager.Tools.Internal
+{
+    internal class SecretsStore
+    {
+        private readonly string _secretsFilePath;
+        private readonly ILogger _logger;
+        private IDictionary<string, string> _secrets;
+        public SecretsStore(string userSecretsId, ILogger logger)
+        {
+            if (userSecretsId == null)
+            {
+                throw new ArgumentNullException(nameof(userSecretsId));
+            }
+
+            _logger = logger;
+
+            _secretsFilePath = PathHelper.GetSecretsPathFromSecretsId(userSecretsId);
+            _logger.LogDebug(Resources.Message_Secret_File_Path, _secretsFilePath);
+
+            {
```

Comment:
define the scope of the workaround. Same could be done w/ a comment block instead.